manager/src/main/resources/application.properties

```diff
@@ -19,6 +19,7 @@ quarkus.flyway.migrate-at-start=true
 quarkus.flyway.validate-on-migrate=true
 quarkus.flyway.ignore-missing-migrations=true
 quarkus.flyway.create-schemas=true
+quarkus.flyway.placeholders.shard-id=${SHARD_ID}
```

Comment:
Don't we need to update `kustomize` scripts to set this value (both `minikube` and `prod`)?

Reply:
We have already the `SHARD_ID` there. It was used for the shard api authentication